### DISCUSSION TITLE
Nulls to control nodes from paths

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#40] [Ideas]
It will be nice if there's some way to link and control individually, the points of some path, with nulls.

**Comment by AlexKiryanov** [comment]
Good afternoon, yes this would be a very nice feature.

**Comment by rodlie** [comment]
That's would be a nice feature, but I would not limit it to just nulls, each path should have proper transform controls and expressions.

Note that I don't have time to work on this for a while (fixes has higher priority than features).

**Comment by AlexKiryanov** [comment]
Good afternoon Thanks for the answer!
